### DISCUSSION TITLE
Welcome to react-native-maps Discussions!

### DISCUSSION THREAD

**Discussion by monholm** [react-native-maps/react-native-maps#4275] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ajith-mohan-dev** [comment]
In iOS, the app crashes when we increase the zoom level.

Could someone update How to increase the zoom level without crashing?

**Comment by anieshka** [comment]
Hi, Is there a way to display a particular country using this? Is there also a way to limit the map to a specific area so that panning and zooming are contained inside that area and you can't zoom out of that specific area? I want to limit what area the user can see on the map. Does anyone have an idea?
